app/upgrades.go

```diff
@@ -36,16 +37,17 @@ var handlers = map[string]appUpgrade{
 	// TODO - Add new upgrade definitions here.
 }
 
+// CustomUpgradeStoreLoader provides upgrade handlers for store and application module upgrades at specified versions
 func CustomUpgradeStoreLoader(app *App, info storetypes.UpgradeInfo) baseapp.StoreLoader {
 	// Register all explicit appUpgrades
-	for name, upgrade := range handlers {
+	for name := range handlers {
 		// If the handler has been defined, add it here, otherwise, use no-op.
 		var handler upgradetypes.UpgradeHandler
-		if upgrade.Handler == nil {
+		if handlers[name].Handler == nil {
```

Comment:
why change this? I was fine before.

Reply:
the linter job is failing ... misread that reported issue, subsequent fix is the actual required update.